exporter/clickhousetracesexporter/clickhouse_exporter.go

```diff
@@ -388,3 +416,212 @@ func (s *storage) Shutdown(_ context.Context) error {
 	}
 	return nil
 }
+
+func extractSpanAttributesFromSpanIndex(span *Span) []SpanAttribute {
+	spanAttributes := []SpanAttribute{}
+	spanAttributes = append(spanAttributes, SpanAttribute{
+		Key:         "traceId",
+		TagType:     "tag",
+		IsColumn:    true,
+		DataType:    "string",
+		StringValue: span.TraceId,
+	})
+	spanAttributes = append(spanAttributes, SpanAttribute{
+		Key:         "spanId",
+		TagType:     "tag",
+		IsColumn:    true,
+		DataType:    "string",
+		StringValue: span.SpanId,
+	})
+	spanAttributes = append(spanAttributes, SpanAttribute{
+		Key:         "parentSpanId",
+		TagType:     "tag",
+		IsColumn:    true,
+		DataType:    "string",
+		StringValue: span.ParentSpanId,
+	})
+	spanAttributes = append(spanAttributes, SpanAttribute{
+		Key:         "name",
+		TagType:     "tag",
```

Comment:
Some of these are (`name`, `spanID`) neither from tag map nor resource map but attributed to "tag". You may want to distinguish them may be in another PR later.

Reply:
Yes we have to introduce new TagType, this will be a change in frontend and query service both.
Let's skip this for now.